karaf-features/src/main/resources/features.xml

```diff
@@ -238,14 +238,14 @@
         <bundle>wrap:mvn:org.rocksdb/rocksdbjni/5.7.3$Export-Package=org.rocksdb;version=5.7.3,org.rocksdb.util;version=5.7.3</bundle>
     </feature>
 
-    <feature name="kafka-streams" description="Kafka Streams" version="2.0.0">
+    <feature name="kafka-streams" description="Kafka Streams" version="${kafka.version}">
         <feature version="5.7.3" dependency="true">rocksdb</feature>
-        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-streams/2.0.0_1</bundle>
-        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-clients/2.0.0_1</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-streams/${kafka.bundle.version}</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-clients/${kafka.bundle.version}</bundle>
         <bundle dependency="true">mvn:org.lz4/lz4-java/1.4.1</bundle>
```

Comment:
We should also update the versions on the transient dependencies to match those use by 2.2.0. For example, 2.2.0 now depends on `lz4-java` 1.5.0.

Reply:
Done